_posts/2016-01-12-StingrayWin.md

```diff
@@ -22,16 +22,17 @@ tags:
 
 
 > The record reflects that a pen register is a device that records the numbers dialed by a particular telephone, and a trap and trace device records the incoming numbers to a telephone. Further, the record reflects that IMSI Catchers, also known as cell site simulators or stingrays, can capture a cell phones unique serial number, its location, and the content of calls, text messages, and web pages visited. Because IMSI catchers’ capabilities are broader, it is improper to equate them to and treat them as pen registers and trap and trace devices. Thus, to the extent that orders address technology other than pen registers and trap and trace devices, they are not exempt under section based on 18 USC 3123(d)(1).
 
-The City of Chicago presented additional arguments that releasing information of how Stingrays are deployed would harm national security. As we've seen since 9/11, the State is quick to drawn on "national security" as a justification for a wide-range of mass surveillance programs, from PRISM to social media monitoring to the war on drugs. In support of the Chicago Police Department's argument was an affidavit from FBI agent Morrison who in fact has submitted the *exact same affidavit* in various other cases. Judge Kennedy also rejected this argument under *Day*, as quoted in her opinion "affidavits will not suffice to prove an exemption based on statements that are 'conclusory, merely recite statutory standards, and are too vague or sweeping.'" (page 14) In essence, this was a boiler-plate affidavit and inadmissible as to how Chicago specifically deploys Stingrays. 
+The City of Chicago presented additional arguments that releasing information of how Stingrays are deployed would harm national security. As we've seen since 9/11, the State is quick to draw on "national security" as a justification for a wide-range of mass surveillance programs, from PRISM and social media monitoring to the war on drugs. In support of the Chicago Police Department's argument was an affidavit from FBI agent Morrison who in fact has submitted the *exact same affidavit* in various other cases. Judge Kennedy also rejected this argument under *Day*, as quoted in her opinion "affidavits will not suffice to prove an exemption based on statements that are 'conclusory, merely recite statutory standards, and are too vague or sweeping.'" (page 14) In essence, this was a boiler-plate affidavit and inadmissible as to how Chicago specifically deploys Stingrays. 
```

Comment:
s/wide-range/wide range/; s/as quoted in her opinion/writing in her opinion that/; s/inadmissable/inapplicable/;